internal/config/mount_config.go

```diff
@@ -174,21 +174,18 @@ type LogRotateConfig struct {
 	Compress        bool `yaml:"compress"`
 }
 
-func DefaultLogRotateConfig() LogRotateConfig {
-	return LogRotateConfig{
-		MaxFileSizeMB:   defaultMaxFileSizeMB,
```

Comment:
Nit: shall we revert the changes made to this and yaml_parser file?
If not, we should remove the unused constants like "defaultMaxFileSizeMB"